simapp/simd/cmd/genaccounts_test.go

```diff
@@ -25,28 +28,39 @@ var testMbm = module.NewBasicManager(genutil.AppModuleBasic{})
 func TestAddGenesisAccountCmd(t *testing.T) {
 	_, _, addr1 := testdata.KeyTestPubAddr()
 	tests := []struct {
-		name      string
-		addr      string
-		denom     string
-		expectErr bool
+		name        string
+		addr        string
+		denom       string
+		withKeyring bool
```

Comment:
can we always setup the keyring in the context and remove the parameter from here?

Reply:
no - it would cause the `invalid address` case to succeed when we want failure